src/libraries/System.Drawing.Common/ref/System.Drawing.Common.cs

```diff
@@ -2187,7 +2187,7 @@ public sealed partial class FrameDimension
         public static System.Drawing.Imaging.FrameDimension Page { get { throw null; } }
         public static System.Drawing.Imaging.FrameDimension Resolution { get { throw null; } }
         public static System.Drawing.Imaging.FrameDimension Time { get { throw null; } }
-        public override bool Equals(object o) { throw null; }
+        public override bool Equals(object? o) { throw null; }
         public override int GetHashCode() { throw null; }
         public override string ToString() { throw null; }
     }
```

Comment:
Below row 2221 - 2223 overloads for `SetColorMatrices(System.Drawing.Imaging.ColorMatrix newColorMatrix, System.Drawing.Imaging.ColorMatrix grayMatrix)` allows null  `newColorMatrix, grayMatrix`. Might be not intentional for `newColorMatrix` but i guess `grayMatrix` could be null

Reply:
Yep, the native method documentation seems to imply that both can be null.